### PR TITLE
Add linting rule to ensure relative imports always have a file extension

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Review by notlee — APPROVED** [review]
let's add a test for this too :) ... in obt :(

File: config.js [hunk 1]
```diff
@@ -10,7 +10,9 @@ module.exports = {
 		"browser": true,
 		"node": true
 	},
+	"plugins": ["import"],
 	"rules": {
+		"import/extensions": [2, "ignorePackages"],
```
Comment (JakeChampion) [inline config.js:15]:
```suggestion
		"import/extensions": [1, "ignorePackages"],
```

File: config.js [hunk 2]
```diff
@@ -10,7 +10,9 @@ module.exports = {
 		"browser": true,
 		"node": true
 	},
+	"plugins": ["import"],
 	"rules": {
+		"import/extensions": [1, "ignorePackages"],
```
Comment (JakeChampion) [inline config.js:15]:
```suggestion
		"import/extensions": ["warn", "ignorePackages"],
```

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v1.1.0 :tada:

**Pull request merged**